src/pagesComponents/subsections/SubsectionForm.tsx

```diff
@@ -117,13 +124,33 @@ function SubsectionFormWithQuery<S extends z.ZodType<any, any>>({
           Baulastträger verwalten…
         </LinkWithFormDirtyConfirm>
       </div>
-
+      <div className="flex items-end gap-5">
+        <LabeledSelect
+          name="subsubsectionStatusId"
+          label="Status"
+          optional
+          options={subsubsectionStatusOptions}
+          outerProps={{ className: "grow" }}
+        />
+        <LinkWithFormDirtyConfirm
+          href={Routes.SubsubsectionStatussPage({ projectSlug })}
+          className="py-2"
+        >
+          Status verwalten…
+        </LinkWithFormDirtyConfirm>
+      </div>
       <LabeledSelect
         name="managerId"
         label="Projektleiter:in"
         optional
         options={getUserSelectOptions(users)}
       />
+      <LabeledTextField
+        type="date"
+        name="estimatedCompletionDate"
+        label="Datum geplante Fertigstellung"
```

Comment:
```suggestion
        label="Datum geplante Fertigstellung"
        hint="Format: <Jahr>-<Monat>, beispielsweise "2026-03"; Wert muss in ein Datum umgewandelt werden können"
```

Der zweite Teil ist IMO der problematische, siehe oben. Ich würde hier die Umwandlung in ein Datum weglassen und in der Website dann den Text ausgeben wie er hier geschrieben ist.

Reply:
@tordans Verstehe deinen Punkt. Wir haben ein Feld mit dem gleichen Namen auf der Subsubsection Ebene - ich habe mich daran orientiert, es könnte sonst auch irgendwie verwirrend sein..? Andererseits sollen wir die Subsubsections ja jett gedanklich ausblenden.